src/iadmin.cpp

```diff
@@ -2675,6 +2677,13 @@ usage( char *subOpt ) {
         "Set the specified OPTION_NAME to OPTION_VALUE for the specified",
         "NAMESPACE in the local zone in R_GRID_CONFIGURATION. If NAMESPACE or",
         "OPTION_NAME is not valid, an error is returned.",
+        " ",
+        "The grid configuration NAMESPACEs and their respective OPTION_NAMEs for",
+        "set_grid_configuration are:",
+        "    authentication",
+        "        password_extend_lifetime",
+        "        password_max_time",
+        "        password_min_time",
```

Comment:
Do you feel users will understand this structure?
Should we provide an example or explain the structure somewhere so people know how to use it?

Reply:
Can always just make a table, I suppose...
```
NAMESPACE             OPTION_NAME
authentication        password_extend_lifetime
authentication        password_max_time
authentication        password_min_time
```

Reply:
much easier to read / understand i think - yes.

Reply:
Done.

Reply:
> much easier to read / understand i think - yes.

I agree.